packages/DB/routes/CustomersRoutes.js

```diff
@@ -30,18 +31,70 @@ router.post("/registration", async (req, res) => {
     const customer = await Customers.findOne({
       login: req.body.login,
     });
+    const allCustomers = await Customers.find();
     if (customer) {
-      res.send({ status: false, message: "this login already exists" });
-    } else {
+      res.send({ registrationMessage: "this login already exists" });
+    }
+    if (!customer) {
       const customer = new Customers(req.body);
       customer.password = bcrypt.hashSync(req.body.password, 10);
       customer.token = jwt.sign({ _id: customer._id }, tokenKey);
       await customer.save();
-      res.send({ status: true, message: "new customer is registered" });
+      if (allCustomers.length === 0) customer.isAdmin = true;
+      await customer.save();
+      const customers = await Customers.find().sort({ login: 1, isAdmin: 1 });
+      res.send({ customers: customers });
     }
   } catch (error) {
     console.log({ message: error });
+    res.status(500).send("something goes wrong with server");
   }
 });
 
+const auth = async (req, res, next) => {
+  const token = (req.headers.authorization || "").replace("Bearer ", "");
+  if (token) {
+    const id = jwt.verify(token, tokenKey);
+    const user = await Customers.findOne({ _id: id._id });
+    if (user.isAdmin) {
+      next();
+    } else res.status(403).send("Not admin");
+  }
```

Comment:
Should we send some response message over there, in case if no token?

Reply:
Now it's not necessary